Tests/SwiftSyntaxParserTest/TokenTest.swift

```diff
@@ -1,5 +1,5 @@
 import XCTest
-import SwiftSyntax
+@testable import SwiftSyntax
```

Comment:
I would very much like to avoid using `@testable`. In CI we currently only build SwiftSyntax in Release mode with testability disabled. If we require `@testable` we need to build SwiftSyntax twice (once as part of the “build” phase without testability and once for the “test” phase with testability). 

The paradigm we are using in SourceKit-LSP is to add a computed underscored property. I would suggest we add a new TestSupport.swift file in the SwiftSyntax target with
```swift
extension TokenKind {
  /// *Public for testing only*
  public var _text: String {
    return text
  }
}
```